packages/remix-dev/cli/migrate/jscodeshift.ts

```diff
@@ -1,60 +1,51 @@
-import { sync as execaSync } from "execa";
+// @ts-ignore `@types/jscodeshift` doesn't have types for this
```

Comment:
Can we use `@ts-expect-error` here instead so we are alerted if/when `@types/jscodeshift` does include types for the runner?

```suggestion
// @ts-expect-error `@types/jscodeshift` doesn't have types for this
```

Reply:
Also I figured out how to add types locally for a 3rd party package, so we could do so for this:


```ts
// types/jscodeshift.d.ts
declare module "jscodeshift/src/Runner" { // I think this will work even though there are `/` in the module specifier.
  interface Options {...} // TODO
  export const run: (transformPath: string, files: string[], options: Options) => Promise<{ error: number }>;
}
```

Reply:
I'll create a PR to DT to fix these types.
Once it's merged, I'll update and remove the `@ts-expect-error`